src/services/user/UsersServices.ts

```diff
@@ -0,0 +1,83 @@
+import { MongoModel } from '@tablerise/database-management';
+import { Logger } from 'src/types/Logger';
+import ValidateData from 'src/support/helpers/ValidateData';
+import { RegisterUserPayload, RegisterUserResponse } from 'src/types/Response';
+import { HttpStatusCode } from 'src/support/helpers/HttpStatusCode';
+import { SchemasUserType } from 'src/schemas';
+import { UserDetail } from 'src/schemas/user/userDetailsValidationSchema';
+import { User } from 'src/schemas/user/usersValidationSchema';
+
+export default class RegisterServices {
+    constructor(
+        private readonly _model: MongoModel<User>,
+        private readonly _modelDetails: MongoModel<UserDetail>,
+        private readonly _logger: Logger,
+        private readonly _validate: ValidateData,
+        private readonly _schema: SchemasUserType // private readonly _cryptographer: any
```

Comment:
esse comentário é necessário?
```suggestion
        private readonly _schema: SchemasUserType,
        private readonly _cryptographer: any
```